src/extension-runners/chromium.js

```diff
@@ -151,12 +151,41 @@ export class ChromiumExtensionRunner {
     });
   }
 
-  wssBroadcast(data: Object) {
-    for (const client of this.wss.clients) {
-      if (client.readyState === WebSocket.OPEN) {
-        client.send(JSON.stringify(data));
+  async wssBroadcast(data: Object) {
+    return new Promise((resolve) => {
+      const clients = this.wss ? new Set(this.wss.clients) : new Set();
+
+      const webExtReloadComplete = async (message) => {
+        const msg = JSON.parse(message);
+
+        if (msg.type === 'webExtReloadExtensionComplete') {
+          for (const client of clients) {
+            client.off('message', webExtReloadComplete);
+          }
+          resolve();
+        }
+      };
+
+      for (const client of clients) {
+        if (client.readyState === WebSocket.OPEN) {
+          client.on('message', webExtReloadComplete);
+          const cleanWebExtReloadComplete = () => {
```

Comment:
Move this to before `if (msg.type === 'webExtReloadExtensionComplete') {` and use a function expression instead of an arrow funtion. Then you can use `let client = this;`.

By doing so, you will be able to remove the `'close'` listener inside the `if (msg.type === 'webExtReloadExtensionComplete') {` handler as well (which is the reason why I wrote this comment in the first place).